packages/suite/src/components/wallet/CoinjoinSummary/CoinJoinCustomStrategy.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import { Translation } from '@suite-components';
+import { Button, P, Switch, variables } from '@trezor/components';
+import { CoinjoinSessionDetail } from './CoinjoinSessionDetail';
+
+const Row = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const TopRow = styled(Row)`
+    align-items: center;
+    margin-bottom: 20px;
+`;
+
+const MiddleRow = styled(Row)`
+    gap: 24px;
+    margin-bottom: 8px;
+`;
+
+const BottomRow = styled(Row)`
+    gap: 12px;
+    margin-top: 16px;
+`;
+
+const Heading = styled.div`
+    font-size: ${variables.FONT_SIZE.H3};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const SetupHeading = styled(Heading)`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+const Fee = styled(Heading)`
+    color: ${({ theme }) => theme.TYPE_GREEN};
+`;
+
+const Subheading = styled.div`
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    margin-bottom: 3px;
+`;
+
+const Text = styled(P)`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const MiningFeeText = styled(Text)`
+    margin-top: 12px;
+`;
+
+const LeftMark = styled(Text)`
```

Comment:
`font-size: ${variables.FONT_SIZE.TINY};` for all

Reply:
added in a95316944db370e685cffa2d264cb5f2cc78a1e1. The `Checkbox` component could be improved a bit, but not in this PR.